### PR TITLE
include BSD license text

### PR DESCRIPTION
**Pull request by danc86**

### CODE REVIEW

**Comment by alexdutton** [comment]
Thank you for this. Can you replace the copyright owner name with "University of Oxford", please, and then I can merge it. (It was written in the course of my employment, so it's theirs :blush:)

**Comment by danc86** [comment]
Sure... commit ac60cf7 is amended to say "University of Oxford".

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Copyright (c) 2016 University of Oxford.
+All rights reserved.
+
+Redistribution and use in source and binary forms are permitted
+provided that the above copyright notice and this paragraph are
+duplicated in all such forms and that any documentation,
+advertising materials, and other materials related to such
+distribution and use acknowledge that the software was developed
+by the Alexander Dutton. The name of the Alexander Dutton may not
```
Comment (danc86) [inline LICENSE]: Oops, better fix these ones too :-)

**Comment by danc86** [comment]
One more try, commit 8557526.

**Pull request merged**